xbmc/cores/VideoPlayer/DVDSubtitles/DVDSubtitlesLibass.cpp

```diff
@@ -420,7 +420,7 @@ void CDVDSubtitlesLibass::ApplyStyle(const std::shared_ptr<struct style>& subSty
     style->SecondaryColour = ConvColor(COLOR::BLACK);
 
     // Configure the effects
-    double lineSpacing = 0.0;
+    double lineSpacing = subStyle->lineSpacing * scale;
```

Comment:
this must be changed.
you have changed scaling by using `scale` and this is wrong, with overrided fonts will cause a wrong scaling.

this has more sense change as:
```c++
    double lineSpacing;
    if (subStyle->assOverrideFont)
      lineSpacing = subStyle->lineSpacing * scaleDefault;
    else
      lineSpacing = subStyle->lineSpacing * scale;
```
and move this code just above the `ass_set_line_spacing` call, since lineSpacing ver is never used before it